stable/xray/values.yaml

```diff
@@ -274,10 +282,40 @@ rabbitmq:
     # existingErlangSecret: <name-of-existing-secret>
   service:
     port: 5672
+  external:
+    rabbitmqUrl:
+    rabbitmqUsername:
+    rabbitmqPassword:
+    rabbitmqErlangCookie:
   persistence:
     enabled: true
     accessMode: ReadWriteOnce
     size: 20Gi
+  loadDefinition:
+    enabled: true
+    existingSecret: "xray-rabbitmq-load-definition"
```

Comment:
This should be empty by default

Reply:
This is a default configuration .This is needed for rabbitmq-repliation issue when replicaCount is more than 1

Reply:
I don't understand. Where does this secret come from? How do you expect it to exist?

Reply:
More Info - https://github.com/bitnami/charts/tree/master/bitnami/rabbitmq#load-definitions
https://github.com/bitnami/charts/blob/master/bitnami/rabbitmq/templates/secrets.yaml#L25